docs/whats-new.asciidoc

```diff
@@ -1,74 +1,78 @@
+As an optional technical preview, you can now {security-guide}/alerts-ui-monitor.html[sort all columns]. To enable the technical preview, switch the *Technical preview* toggle on. If you experience performance issues, you can turn this setting off.
 
-[discrete]
-[[index-updates-8.0]]
-== Index and privilege updates
+[role="screenshot"]
+image::whats-new/images/8.1/column-sort.gif[]
 
-*Index changes*
+The following {security-guide}/rules-ui-management.html[new bulk actions] can be performed on multiple rules simultaneously:
 
-* The system index for detection alerts has been renamed from `.siem-signals-<space-id>` to `.alerts-security.alerts-<space-id>`. Users who use the detections feature require new privileges for these indices. Users also need new privileges to {security-guide}/alerts-ui-manage.html[view alerts] and {security-guide}/rules-ui-create.html#preview-rules[preview rules]. Refer to {security-guide}/detections-permissions-section.html[Detections prerequisites and requirements] for a breakdown of what privileges are required to access specific detection features.
-* The schema used for alert documents in {elastic-sec} has changed. Users that access documents in the `.siem-signals-<space-id>` indices, such as customer dashboards, will need to update their API queries and scripts to operate correctly on the new 8.x alert documents. Refer to the new {security-guide}/alert-schema.html[alert schema] to view legacy 7.x field names and the updated field names for 8.x, and review {security-guide}/query-alert-indices.html[best practices on how to query alert indices].
-* You can now grant feature privileges for Cases separately from privileges for {elastic-sec}. Refer to {security-guide}/case-permissions.html[Cases prerequisites] for more information.
-* New Full Disk Access approval is needed when deploying {elastic-endpoint} on macOS. Refer to {security-guide}/deploy-elastic-endpoint.html[Install Elastic Endpoint manually] to view installation requirements.
+* Add index patterns
+* Delete index patterns
+* Add tags
+* Delete tags
 
+Select the *Bulk actions* menu to view all available bulk options.
 
+[role="screenshot"]
+image::whats-new/images/8.1/bulk-actions.png[]
+
+*Event filters and host isolation exceptions can be applied per policy*
+
+Users with a https://www.elastic.co/pricing/[Platinum or Enterprise license] can apply new or existing {security-guide}/event-filters.html[event filters] and {security-guide}/host-isolation-exceptions.html[host isolation exceptions] to a specific integration policy instead of globally. This allows you to configure settings that may be unique to a specific group within your network.
```

Comment:
```suggestion
Users with a https://www.elastic.co/pricing/[Platinum or Enterprise license] can now apply new or existing {security-guide}/event-filters.html[event filters] and {security-guide}/host-isolation-exceptions.html[host isolation exceptions] to a specific integration policy instead of globally. This allows you to configure settings that may be unique to a specific group within your network.
```